controllers/topics.controller.js

```diff
@@ -0,0 +1,11 @@
+const { findTopics } = require('../models/topics.models')
+
+exports.getTopics = (req, res, next) => {
+    findTopics(req.params)
```

Comment:
We don't need to put the req.params into the model here. Have a think about what you're getting back from the model. Are you passing anything to it from the request?